docs/applications/configure-dns-service.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Configure a Domain Name Server (DNS) Service
+intro: Your web application users may benefit from using a domain name instead of an IP address to access your services. Configure a Domain Name Server (DNS) service to manage the routing to your website, redirecting the user accessing the human readable domain to your server IP Address. Some examples of DNS services that can be used include FreeDNS, AWS route 53, cloudflare DNS.
+links:
+    overview:
+    quickstart:
+    previous:
+        - /docs/applications/add-ssl-certificate
+    next:
+        - /docs/roles/add-role
+    guides:
+    related:
+    featured:
+---
+
+1. On Devopness, navigate to a project then select an environment
+1. Find the `Servers` card
+1. Click the `View` in the `Servers` card to see a list of existing `Servers`
+1. In the list of servers, find the server you want to connect and copy the server' IP Address
+    > The ip address should be a number separated by dots, e.g: 41.234.56.78
```

Comment:
If the user will find and copy the IP address, why the docs are using imperative words such as "should" for something that is already defined and users cannot change it at this stage?
As discussed in previous PRs, please re-consider if you really need to explain attributes in the middle of a sequence of steps that should be easy to follow and lead to success.

Reply:
In this example, explaining what is the format of an IP Address helps the user to locate it in the page.

Also, the instructions used here follow the same steps approved in ssh into server.

![image](https://github.com/devopness/devopness/assets/33532177/a12a8796-01f1-4a62-afdb-9600646d1c66)

Reply:
I suggest replacing "should be" to "is", because there is no action expected from the user here.

Reply:
@jfoliveira updated